client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -391,12 +402,14 @@ def imprint(node, data, update=False):
 
     # Add new parm templates
     if new_parm_templates:
-        parm_folder = parm_group.findFolder("Extra")
+        parm_folder = (
+            parm_group.findFolder("AYON") or parm_group.findFolder("Extra")
+        )
 
         # if folder doesn't exist yet, create one and append to it,
         # else append to existing one
         if not parm_folder:
-            parm_folder = hou.FolderParmTemplate("folder", "Extra")
+            parm_folder = hou.FolderParmTemplate("folder", "AYON")
```

Comment:
Wouldn't this break backwards compatibility with older scenes?

Reply:
As far as I know - it shouldn't. The folder isn't part of the unique identifier of the attribute - the attribute name itself is. As such, no matter what folder it is in or we move it to... it'll behave the same and is accessible the same - it's just in a visually different tab for the user after this PR.

Of course, something to test.